scripts/setup-volume-testnet.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+set -x
+
+if ! which jq > /dev/null; then
+  echo 'command jq not found, please install jq'
+  exit 1
+fi
+
+CHAIN_ID="paloma"
+
+palomad init my_validator --chain-id "$CHAIN_ID"
+
+pushd ~/.paloma/config/
+sed -i 's/^keyring-backend = ".*"/keyring-backend = "test"/' client.toml
+sed -i 's/^minimum-gas-prices = ".*"/minimum-gas-prices = "0.001grain"/' app.toml
+sed -i 's/^laddr = ".*:26657"/laddr = "tcp:\/\/0.0.0.0:26657"/' config.toml
+jq ".chain_id = \"${CHAIN_ID}\"" genesis.json > genesis.json.tmp && mv genesis.json.tmp genesis.json
+jq 'walk(if type == "string" and .. == "stake" then "grain" else . end)' genesis.json > genesis.json.tmp && mv genesis.json.tmp genesis.json
+popd
+
+VALIDATOR_STAKE_AMOUNT=100000000grain
+init_validator() {
+  name="$1"
+  address="$2"
+  
+  palomad add-genesis-account "$address" 250000000grain
+  palomad gentx "$name" "$VALIDATOR_STAKE_AMOUNT" --chain-id "$CHAIN_ID"
+}
+
+# Faucet
+palomad add-genesis-account paloma1fgy8ghc70swdzzpa4xpk544xs7q8xh3zfkzal2 4000000000grain
```

Comment:
Do we want less in the faucet and any excess (up to 5B) stored in a 6th account?